tests/framework/unit_tests/DataObjects/TestDataSet.py

```diff
@@ -633,8 +633,8 @@ def formatRealization(rlz):
     checkTrue('CSV var {}'.format(var),bool((dataCSV._data[var] == data._data[var]).prod()))
 
 # clean up temp files
-os.remove(csvname+'.csv')
-os.remove(csvname+'.xml')
+#os.remove(csvname+'.csv')
+#os.remove(csvname+'.xml')
```

Comment:
we do not remove them anymore? 
Probably we should just check if they exists and in case, remove

Reply:
We can remove them; this was a debugging exercise that was not reverted.  Fixed.